### ISSUE TITLE
Remove the "(forced)" part in the label for forced closed captions

### ISSUE THREAD

**Issue by marcosthejew** [shaka-project/shaka-player#5685]
**Have you read the [Tutorials](http://shaka-player-demo.appspot.com/docs/api/tutorial-welcome.html)?**
Yes.

**Have you read the [FAQ](https://bit.ly/ShakaFAQ) and checked for duplicate open issues?**
Yes.

**What version of Shaka Player are you using?**
4.3.7

**Please ask your question**
Is there any way to remove the (Forced) part of a label from a Closed Captions instance when the call to `addTextTrackAsync `has the parameter `force==true`? We don't want our users to read that the captions are "forced". Thanks.
<img width="202" alt="image" src="https://github.com/shaka-project/shaka-player/assets/1500967/0dc74b0c-a734-4cee-9889-fc6d6466d3fc">

**Comment by absidue** [comment]
You could hide the captions selector, through the UI configuration.

https://shaka-player-demo.appspot.com/docs/api/tutorial-ui-customization.html

**Comment by marcosthejew** [comment]
Thanks @absidue but if I do that the captions won't show on iOS

**Comment by marcosthejew** [comment]
Even when calling: `player.selectTextTrack(track,true)` and `player.setTextTrackVisibility(true)` the subtitles do not show on iOS. Moreover what if I wanted to add multiple text tracks in different languages for users to choose from? They would not be able to select the track they wanted as they don't appear listed on the iOS default Video Controls

**Comment by absidue** [comment]
I'm confused why you would use forced captions but want the user to have control over which caption is used, the whole point of using forced captions is that you don't want the user to have control.

**Comment by marcosthejew** [comment]
@absidue For some reason I'm getting a "code 2012:External text tracks cannot be added in src= because native platform doesn't support it." for iOS if I don't set the parameter force to true. This happens in the project I'm working on, but I cannot replicate it using the demo.

Thanks @avelad is there any solution for captions not showing when going full screen on iOS?

**Comment by marcosthejew** [comment]
That's what we are aiming for in the future, but for now, we will need to modify a lot of manifests.

**Comment by absidue** [comment]
@marcosthejew what value are you passing for `kind`? Make sure that it's a valid one `captions` or `subtitles` https://developer.mozilla.org/en-US/docs/Web/API/TextTrack/kind

**Comment by marcosthejew** [comment]
Thanks @absidue and @avelad , upgrading to mux.js@7.0.0 in the requirements fixed the issue

**Comment by marcosthejew** [comment]
Sorry, spoke too soon. The captions load fine but with an addition unrecognized() track. here is the code:

shaka-test.html : 
`<!DOCTYPE html>
<html>
  <head>
    
    <script src=https://cdnjs.cloudflare.com/ajax/libs/mux.js/7.0.0/mux.min.js></script>
    <script src="https://ajax.googleapis.com/ajax/libs/shaka-player/4.3.7/shaka-player.ui.js"></script>
    
    <script src="myapp.js"></script>
    <link rel="stylesheet" type="text/css" href="dist/controls.css">
  </head>
  <body>
    <div id="videoContainer" style="width:640px; height:360px;">
    <video id="video"
           poster="https://shaka-player-demo.appspot.com/assets/poster.jpg"
           controls autoplay></video>
          </div>
  </body>
</html>
`
myapp.js
`const manifestUri =
    'https://demo.unified-streaming.com/k8s/features/stable/video/tears-of-steel/tears-of-steel.ism/.m3u8';

function initApp() {
  // Install built-in polyfills to patch browser incompatibilities.
  shaka.polyfill.installAll();

  // Check to see if the browser supports the basic APIs Shaka needs.
  if (shaka.Player.isBrowserSupported()) {
    // Everything looks good!
    initPlayer();
  } else {
    // This browser does not have the minimum set of APIs we need.
    console.error('Browser not supported!');
  }
}

async function initPlayer() {
  // Create a Player instance.
  const video = document.getElementById('video');
  const player = new shaka.Player(video);

  // Attach player to the window to make it easy to access in the JS console.
  window.player = player;
  //debugger;
  // Listen for error events.
  player.addEventListener('error', onErrorEvent);


  // Try to load a manifest.
  // This is an asynchronous process.
  try {
    await player.load(manifestUri);
    // This runs if the asynchronous load is successful.
    console.log('The video has now been loaded!');
    shaka.polyfill.installAll();
    shaka.polyfill.PatchedMediaKeysApple.install();
    const videoContainer = document.getElementById('videoContainer');
    const ui = new shaka.ui.Overlay(player, videoContainer, player.getMediaElement());
    const config = {
            controlPanelElements: [
                'play_pause',
                'spacer',
                'time_and_duration',
                'volume',
                'mute',
                'picture_in_picture',
                'fullscreen',
                'captions',
            ],
        };
        ui.configure(config);
        ui.setEnabled(true);

    player.addTextTrackAsync('https://raw.githubusercontent.com/1c7/vtt-test-file/master/vtt%20files/2.%20No%20Index%20Number%2C%20download%20using%20youtube-dl.vtt', 'en', 'captions', 'text/vtt').then(function(track) {
    }).catch(function error(error){
      console.log(error);
  });
  
  } catch (e) {
    // onError is executed if the asynchronous load fails.
    onError(e);
  }
}

function onErrorEvent(event) {
  // Extract the shaka.util.Error object from the event.
  onError(event.detail);
}

function onError(error) {
  // Log the error.
  console.error('Error code', error.code, 'object', error);
}

document.addEventListener('DOMContentLoaded', initApp);`
